onnxruntime/test/global_thread_pools/test_inference.cc

```diff
@@ -58,7 +58,7 @@ static void RunSession(OrtAllocator& allocator, Ort::Session& session_object,
 
   OutT* f = output_tensor->GetTensorMutableData<OutT>();
   for (size_t i = 0; i != static_cast<size_t>(5); ++i) {
-    ASSERT_TRUE(abs(values_y[i] - f[i]) < 1e-6);
+    ASSERT_TRUE(std::abs(values_y[i] - f[i]) < 1e-6);
```

Comment:
There is an `ASSERT_NEAR` in gtest.

https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#floating-point-macros